### ISSUE TITLE
Flatnotes as a PWA

### ISSUE THREAD

**Issue by effree** [dullage/flatnotes#124]
I noticed that Flatnotes can be a PWA when using the demo site (in Brave on mobile and on PC is says "Install Flatnotes"), but not when installed as my own app in docker.  This includes, when setting it up for external use with a similar sub-domain structure as the demo website.

**Comment by dullage** [comment]
Hi @effree. I believe an app needs to be served over HTTPS for it to be "installable". If you are serving it over plain HTTP then perhaps this is the problem?

**Comment by effree** [comment]
@dullage thank you for your quick response.  I am serving the application over https, although i suppose there could be a minimum SSL encryption requirement, that i may not have with my basic Let's Encrypt SSL.  Another thing i was thinking, is that i am using Authelia for security, although, i don't think that should interfere.

**Comment by dullage** [comment]
@effree - Another thing to check would be the manifest tab in Chrome dev tools. That might give you a warning. 

Also check your on the latest version of flatnotes as there were fixes to this recently.

**Comment by effree** [comment]
@dullage I am on the latest version of Flatnotes.

I looked at the manifest, and it does have the following:
`Note: id is not specified in the manifest, start_url is used instead. To specify an App Id that matches the current identity, set the id field to / `.

I followed this page and it showed an example of adding /?homescreen=1 to your url, which i did and it actually allows me to install as PWA on my Macbook in Brave.

Doing the same on Android is still not working though.

I compared the demo to mine and they appear to have the same messages.  I wonder if the Demo site and this site, since they are almost identical cause one to be allowed to be PWA and the other is not based on browsing history or cache, because they don't have some sort of unique app_id.
